src/v0/destinations/rakuten/networkHandler.js

```diff
@@ -0,0 +1,102 @@
+const { NetworkError } = require('@rudderstack/integrations-lib');
+const { httpSend } = require('../../../adapters/network');
+const {
+  processAxiosResponse,
+  getDynamicErrorType,
+} = require('../../../adapters/utils/networkUtils');
+const { TAG_NAMES } = require('../../util/tags');
+const { HTTP_STATUS_CODES } = require('../../util/constant');
+
+const DESTINATION = 'RAKUTEN';
+const prepareProxyRequest = (request) => request;
+const proxyRequest = async (request, destType) => {
+  const { endpoint, data, method, params, headers } = prepareProxyRequest(request);
+  const requestOptions = {
+    url: endpoint,
+    data,
+    params,
+    headers,
+    method,
+  };
+  const response = await httpSend(requestOptions, { feature: 'proxy', destType });
+  return response;
+};
+const extractContent = (xmlPayload, tagName) => {
```

Comment:
write unit test for this

Reply:
This is a common function between adobe_analytics and rakuten hence will bring it up global utils in one PR